code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -121,6 +121,7 @@
 			if(MAP_ICE_COLONY)
 				new /obj/item/clothing/suit/storage/snow_suit(src)
 				new /obj/item/clothing/mask/rebreather/scarf(src)
+				new /obj/item/clothing/mask/rebreather(src)
```

Comment:
Instead of spawning both, which generates clutter, you could add it to the list of mask options in the marine role vendors. Both of them, really. In where right now it's only offered a gas mask.

Reply:
This is for engineering lockers-- so it's gotta remain there. I'll add em to the gas mask option however.